Makefile

```diff
@@ -50,11 +53,19 @@ prepare_cpp_project:
 windows-compile:
 	@echo "Compiling..."
 	@$(CASK) $(EMACS) -Q --batch \
-		-l test/windows-bootstrap.el \
+		-l $(WIN-BOOTSTRAP) \
 		-L . -L clients \
 		--eval '(setq byte-compile-error-on-warn t)' \
 		-f batch-byte-compile $(LSP-FILES)
 
+test-pkgs:
+	@echo "Test emacs-lsp packages..."
+	@$(CASK) $(EMACS) -Q --batch \
+		-l $(WIN-BOOTSTRAP) \
+		--eval '(setq byte-compile-error-on-warn t)' \
```

Comment:
can we move that in the test-package.el? Can we structure the code like that:

1. Install lsp-* package dependencies(e. g. lsp-treemacs depends on treemacs)
2. After that set byte-compile-error-on-warn to t and install our packages.

Reply:
You mean by setting `byte-compile-error-on-warn` to `t` only for emacs-lsp packages?

Reply:
yes. I don't want the CI to be red if some of the deps has byte-compilation issue.

Reply:
Any suggestion achieving this? Not sure if there is a clean way to do this. 😕

Reply:
My idea was to have 2 loops of package-install, first will install the deps of the emacs lsp packages, the second will install the actual packages. But I am fine with anything that covers the initial requirements.